src/main/resources/templates/snowflake/snowflake-schema-evolution/no-change.sql.ssp

```diff
@@ -8,4 +8,4 @@
     val destination = configuration.destination.asInstanceOf[Snowflake]
 }#
 
-SELECT 'NO CHANGES in ${destination.stagingDatabase.name}.${destination.stagingDatabase.schema}.${table.destinationName}';
\ No newline at end of file
+SELECT 'INCOMPATIBLE CHANGES in ${destination.stagingDatabase.name}.${destination.stagingDatabase.schema}.${table.destinationName}';
```

Comment:
Do we need two files? One for no changes and one for incompatible? There are cases where are no changes and yet we want a valid makefile so it seems like we want the no-change template.

Reply:
I think we need one template  only for incompatible changes. In tableDiffs only those tables are added which have atleast one change it could be compatible or incompatible. For compatible we already have ALTER stmts. And for incompatible we can use incompatible template file. 
In case a table have no change then that table is not processed during schema evolve. 
Also in case there is no schema changes then tableDiff will be empty. To have a valid Makefile we have added a target  
evolve-schema-all: 
This target won't do anything.
I should rename the template name from no-change.sql.ssp to incompatible-change.sql.ssp

Reply:
> I think we need one template only for incompatible changes
 Do we need a template when there is "no change"

Reply:
Based on below, I think the answer is no.